libass/ass_cache.c

```diff
@@ -40,12 +40,13 @@
 // font cache
 static bool font_key_move(void *dst, void *src)
 {
-    ASS_FontDesc *k = src;
-    if (dst)
-        memcpy(dst, src, sizeof(ASS_FontDesc));
-    else
-        free(k->family);
-    return true;
+    if (!dst)
+        return true;
+
+    ASS_FontDesc *d = dst, *s = dst;
```

Comment:
I’m confused by these two variables.

Reply:
Yes, there is a typo here but it's magically not leading to any error.

Reply:
Yep 😃 The code works equally well as it is now, and with `*s = src`, and with just a single variable. But either of the last two options would have been easier to read.